custom_components/husqvarna_automower/translations/et.json

```diff
@@ -43,33 +43,33 @@
     "application_credentials": {
         "description": "Loo konto [Husqvarna developers portal]({oauth_creds_url}). Ühenda Authentication API ja Automower Connect API. Sea ümbersuunamise URI https://my.home-assistant.io/redirect/oauth. Kui sa pole veel seadistanud My HomeAssistant asukohta, sisesta see URL oma asukohaks {redirect_uri}"
     },
-	"entity": {
-		"sensor": {
-			"problem_list": {
-				"state": {
-					"off": "Väljas",
-					"unknown": "Teadmata",
-					"stopped": "Peatunud",
-					"stopped_in_garden": "Peatunud aias",
-					"not_applicable": "Pole kohalduv",
-					"none": "Puudub",
-					"week_schedule": "Nädala ajakava",
-					"park_override": "Parkimise tühistamine",
-					"sensor": "Ilmataimer",
-					"daily_limit": "Päevane limiit",
-					"fota": "Püsivara uuendamine",
-					"frost": "Härmatis"
-				}
-			},
-			"mode_list":{
-				"state": {
+    "entity": {
```

Comment:
Is there a reason, why you change des number of spaces?
I want to keep the translations and the strings.json all in the same format

Reply:
I noted that some indenting was done with tabs, some with spaces. I did not wat to change the number of spaces, it was probably done by github editor. I'm a bit confused, please disregard this PR alltogether.

Reply:
Ok